from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(1000, 800)
        MainWindow.setAutoFillBackground(False)
        MainWindow.setStyleSheet(
            "background-color: rgb(220, 221, 255);\n"
            "border-top-color: rgb(255, 255, 255);"
        )
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.app_logo = QtWidgets.QLabel(self.centralwidget)
        self.app_logo.setGeometry(QtCore.QRect(270, 20, 451, 131))
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(72)
        font.setBold(True)
        font.setWeight(75)
        self.app_logo.setFont(font)
        self.app_logo.setStyleSheet("color: rgb(255, 203, 99);\n" "")
        self.app_logo.setObjectName("app_logo")
        self.email_input = QtWidgets.QLineEdit(self.centralwidget)
        self.email_input.setGeometry(QtCore.QRect(440, 260, 351, 51))
        font = QtGui.QFont()
        font.setPointSize(18)
        self.email_input.setFont(font)
        self.email_input.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.email_input.setText("")
        self.email_input.setObjectName("email_input")
        self.email_label = QtWidgets.QLabel(self.centralwidget)
        self.email_label.setGeometry(QtCore.QRect(250, 260, 101, 41))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.email_label.setFont(font)
        self.email_label.setObjectName("email_label")
        self.pw_label = QtWidgets.QLabel(self.centralwidget)
        self.pw_label.setGeometry(QtCore.QRect(250, 350, 121, 41))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.pw_label.setFont(font)
        self.pw_label.setObjectName("pw_label")
        self.new_label = QtWidgets.QLabel(self.centralwidget)
        self.new_label.setGeometry(QtCore.QRect(360, 580, 311, 41))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(20)
        font.setBold(True)
        font.setWeight(75)
        self.new_label.setFont(font)
        self.new_label.setObjectName("new_label")
        self.login_button = QtWidgets.QPushButton(self.centralwidget)
        self.login_button.setGeometry(QtCore.QRect(670, 450, 121, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(18)
        font.setBold(True)
        font.setWeight(75)
        self.login_button.setFont(font)
        self.login_button.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.login_button.setObjectName("login_button")
        self.new_button = QtWidgets.QPushButton(self.centralwidget)
        self.new_button.setGeometry(QtCore.QRect(410, 670, 201, 51))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.new_button.setFont(font)
        self.new_button.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.new_button.setObjectName("new_button")
        self.pw_input = QtWidgets.QLineEdit(self.centralwidget)
        self.pw_input.setGeometry(QtCore.QRect(440, 350, 351, 51))
        font = QtGui.QFont()
        font.setPointSize(18)
        self.pw_input.setFont(font)
        self.pw_input.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.pw_input.setText("")
        self.pw_input.setObjectName("pw_input")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 1000, 21))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.app_logo.setText(_translate("MainWindow", "GoldApp"))
        self.email_label.setText(_translate("MainWindow", "Email:"))
        self.pw_label.setText(_translate("MainWindow", "Password:"))
        self.new_label.setText(_translate("MainWindow", "Is it your first time here?"))
        self.login_button.setText(_translate("MainWindow", "Login"))
        self.new_button.setText(_translate("MainWindow", "Make a new account"))


if __name__ == "__main__":
    import sys

    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    ui = Ui_MainWindow()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec_())
